algoliasearch-common/src/main/java/com/algolia/search/Defaults.java

```diff
@@ -19,7 +22,9 @@
   ObjectMapper DEFAULT_OBJECT_MAPPER =
       new ObjectMapper()
           .enable(OBJECT_MAPPER_DEFAULT_FEATURE)
-          .disable(OBJECT_MAPPER_DEFAULT_DESERIALIZATION_FEATURE);
+          .disable(OBJECT_MAPPER_DEFAULT_DESERIALIZATION_FEATURE)
+          .registerModule(new JavaTimeModule())
```

Comment:
careful with this.
Users can override the default object mapper. So the `JavaTimeModule` might not present.

Reply:
I did not know that. What would be the best way to enforce its use then. I've checked extensively and the `JavaTimeModule` is the correct way to format Java8 time related classes. I'd like to stick with this rather that workarounds.